package.json

```diff
@@ -426,6 +426,17 @@
                     "description": "Specifies when to switch between displaying files as a `tree` or `list` based on the number of files in a nesting level in the `GitLens` explorer\nOnly applies when displaying files as `auto`",
                     "scope": "window"
                 },
+                "gitlens.gitExplorer.branches.layout": {
+                    "type": "string",
+                    "default": "list",
+                    "enum": [
+                        "list",
+                        "tree",
+                        "mix-tree"
+                    ],
+                    "description": "Specifies how the `Branches` view will display branches\n `list` - display all branches \n`tree` - organize branch as folder if branch name contains slashes \"\/\"\n `mix-tree` - display branch folders along with normal branch alphabetically",
```

Comment:
Any objection to getting rid of `mix-tree` and just having `tree` behave this way (i.e. sorted naturally)?

Reply:
Or if you'd like to keep the feature, can we change it to a different setting?

Like `gitlens.gitExplorer.branches.sort` = `'alphabetic' | 'foldersFirst' | 'foldersLast'?
or
`gitlens.gitExplorer.branches.folderGrouping` = `'none' | 'start' | 'end'?

Thoughts?

Reply:
I think having only two options (`list` and `tree`) is fine, actually current tree sort implementation (not the `mix-tree` one) calculates more. It runs a recursive function, marking the parent folder as current (`BranchFolderNode#findCurrent`) 

If you think it's okay, I can drop those commits 😄

Reply:
Ok let's drop it and just have the 2 options. Thanks!